src/forwarders.js

```diff
@@ -387,23 +393,40 @@ export default function Forwarders(mpInstance, kitBlocker) {
         }
     };
 
-    this.callSetUserAttributeOnForwarders = function(key, value) {
+    this.onHandleForwarderUserAttributes = function(
```

Comment:
Nit - I feel like this should be `handleForwarderUserAttributes` instead of `onHandleForwarderUserAttributes`.  `on` something is more an effect, almost like a callback (but not quite), or something that is not directly associated with it the verb it is `on`-ing if that makes sense.  So in our kit code, it is `onHandle` or `onUserIdentified` because the intent when I call `mParticle.....setUserAttribute` is to change it in our system...and then the the kit has the downstream effect from the core.  it's a bit nuanced (and perhaps my opinion) but in the core SDK, i don't think we use `on` anything....otherwise we'd see `on-Verb` a lot more.

Reply:
Agreed. The `on` prefix for me is usually associated with UI functionality, which feels misleading in this context.

Reply:
Sounds good, will implement changes, thanks for feedback!

Reply:
@rmi22186 and @alexs-mparticle, I was able to take off one unnecessary `if` statement and make a helper function to reduce the messy lines to check if attribute is filtered as requested, and changed `onHandleForwarderUserAttributes` to `handleForwarderUserAttributes`. Let me know if anything else is needed.